### PR TITLE
Add 'Migrating Configurations' section

### PR DESCRIPTION
**Pull request by AhyoungRyu**
I added "Migrating Configuration" section at the bottom of [http://help.zeppelinhub.com/zeppelin_integration/](http://help.zeppelinhub.com/zeppelin_integration/) page. This section will be reachable with this link when it's deployed(Not deployed yet).
[http://help.zeppelinhub.com/zeppelin_integration/#migrating-configurations](http://help.zeppelinhub.com/zeppelin_integration/#migrating-configurations)
And it'll be shown like below.

screen shot 2016-08-22 at 5 02 38 pm

### CODE REVIEW

**Comment by AhyoungRyu** [comment]
@anthonycorbacho @khalidhuseynov @sejunra Could you review this one?

**Comment by AhyoungRyu** [comment]
As @sejunra said, I made the code block scroll-bar always visible.
screen shot 2016-08-22 at 11 26 55 am

But sadly it doesn't work in Firefox since it doesn't allow the custom scrollbar ㅠ_ㅠ

File: docs/zeppelin_integration.md [hunk 1]
```diff
@@ -95,14 +95,36 @@ export ZEPPELINHUB_API_TOKEN="YOUR_TOKEN_STRING"
...
+If you upgraded Zeppelin-0.5.X to later version, you need to change some configurations in Apache Zeppelin.
+
+1\. Remove `zeppelinhub-integration-*.jar` file under `ZEPPELIN_HOME/lib/` directory.
+```
+$ rm ZEPPELIN_HOME/lib/zeppelinhub-integration-*.jar
+```
+
+2\. Open `ZEPPELIN_HOME/conf/zeppelin-env.sh` and replace `ZEPPELIN_NOTEBOOK_STORAGE` value like below
+```sh
+export ZEPPELIN_NOTEBOOK_STORAGE="org.apache.zeppelin.notebook.repo.VFSNotebookRepo, org.apache.zeppelin.notebook.repo.zeppelinhub.ZeppelinHubRepo"
```
Comment (Leemoonsoo) [inline docs/zeppelin_integration.md]: Does Zeppelin parse classname correctly? there is a space after comma.

And i think it's also good idea to explicitly tell user that fully qualified classname `org.apache.zeppelin.notebook.repo.zeppelinhub.ZeppelinHubRepo` has been changed. Some user may read first few characters "org.apache.zeppelin.notebook.rep..." and might think it's okay to not update.

Comment (AhyoungRyu) [inline docs/zeppelin_integration.md]: @Leemoonsoo Right. Need to remove the space. 
Yeah right that would be better (It's little bit like a 틀린그림찾기 kkkkk)
Let me update it :)

Comment (AhyoungRyu) [inline docs/zeppelin_integration.md]: @Leemoonsoo Regarding the space, it's not a problem i think. As you can see in the above section "Zeppelin-0.6.X or later", there is also an extra space next to the `org.apache.zeppelin.notebook.repo.VFSNotebookRepo,`. I have always copied that code block and it works well.

Comment (khalidhuseynov) [inline docs/zeppelin_integration.md]: @Leemoonsoo space is okay there, it just trims it

**Comment by AhyoungRyu** [comment]
@Leemoonsoo Addressed your feedback and updated the screenshot as well.

**Comment by khalidhuseynov** [comment]
@AhyoungRyu do we need to mention that we expect them to have the following ones from setup of 0.5.x
```
export ZEPPELINHUB_API_ADDRESS="ADDRESS_OF_ZEPPELINHUB_SERVICE" (e.g. https://www.zeppelinhub.com)
export ZEPPELINHUB_API_TOKEN="YOUR_TOKEN_STRING"
```
what do you think, or is it obvious?

**Comment by AhyoungRyu** [comment]
@khalidhuseynov Since i thought they already have those two variables, I didn't mention about them. This way is more obvious i guess. If not, ppl might think "So what is different between `Zeppelin-0.5.x` and `Zeppelin-0.6.x`? it seems almost same". But It's just my opinion. 
While all we already fully perceived those info, users may not. So if we need to mention the two variable, let's add them :)

File: docs/zeppelin_integration.md [hunk 2]
```diff
@@ -95,14 +95,38 @@ export ZEPPELINHUB_API_TOKEN="YOUR_TOKEN_STRING"
...
+<br/>
+# Migrating Configurations
+If you upgraded Zeppelin-0.5.X to later version, you need to change some configurations in Apache Zeppelin.
+
+1\. Remove `zeppelinhub-integration-*.jar` file under `ZEPPELIN_HOME/lib/` directory.
+```
+$ rm ZEPPELIN_HOME/lib/zeppelinhub-integration-*.jar
+```
+
+2\. Open `ZEPPELIN_HOME/conf/zeppelin-env.sh` and replace `ZEPPELIN_NOTEBOOK_STORAGE` value like below
```
Comment (khalidhuseynov) [inline docs/zeppelin_integration.md]: how about the following here @AhyoungRyu @Leemoonsoo :
2. From your Zeppelin-0.5.X installation you should have the following variables already set in your `ZEPPELIN_HOME/conf/zeppelin-env.sh`
```
export ZEPPELIN_NOTEBOOK_STORAGE="org.apache.zeppelin.notebook.repo.VFSNotebookRepo, com.nflabs.zeppelinhub.notebook.repo.ZeppelinHubRepo"
export ZEPPELINHUB_API_ADDRESS="ADDRESS_OF_ZEPPELINHUB_SERVICE" (e.g. https://www.zeppelinhub.com)
export ZEPPELINHUB_API_TOKEN="YOUR_TOKEN_STRING"

# With user key if provided
export ZEPPELINHUB_USER_KEY="YOUR_USER_KEY"
```
Starting from Zeppelin-0.6.0 it is required that the second classname in `ZEPPELIN_NOTEBOOK_STORAGE` variable to be changed from `com.nflabs.zeppelinhub.notebook.repo.ZeppelinHubRepo` into `org.apache.zeppelin.notebook.repo.zeppelinhub.ZeppelinHubRepo`. Thus you need to substitute 
```
export ZEPPELIN_NOTEBOOK_STORAGE="org.apache.zeppelin.notebook.repo.VFSNotebookRepo, com.nflabs.zeppelinhub.notebook.repo.ZeppelinHubRepo"
```
with 
```
export ZEPPELIN_NOTEBOOK_STORAGE="org.apache.zeppelin.notebook.repo.VFSNotebookRepo, org.apache.zeppelin.notebook.repo.zeppelinhub.ZeppelinHubRepo"
```
in your `ZEPPELIN_HOME/conf/zeppelin-env.sh`.

Comment (AhyoungRyu) [inline docs/zeppelin_integration.md]: That's what we want @khalidhuseynov thanks!  I'll update as like you said :)

**Comment by AhyoungRyu** [comment]
@khalidhuseynov Updated the docs again as you said. And the screenshot as well.

**Comment by khalidhuseynov** [comment]
looks good!

**Comment by AhyoungRyu** [comment]
@khalidhuseynov Thx! May i merge this now then?

**Comment by anthonycorbacho** [comment]
@AhyoungRyu you may processed

**Pull request merged**